src/ontology/mondo-edit.obo

```diff
@@ -462878,6 +462873,19 @@ name: neurological muscular channelopathy due to a genetic ryanodine receptor de
 xref: Orphanet:98742 {source="MONDO:equivalentTo"}
 is_a: MONDO:0019119 {source="Orphanet:98742", source="https://orcid.org/0000-0001-5208-3432"} ! muscular channelopathy
 
+[Term]
+id: MONDO:1010000
+name: pythiosis
+def: "A granulomatous disease caused by the aquatic organism pythium insidiosum occurring rarely in humans. It is classified into three forms: ocular, cutaneous, and arterial." [MESH:D058968]
+synonym: "human pythiosis" RELATED [GARD:0011989]
+synonym: "pythium insidiosum infection" RELATED [GARD:0011989]
+xref: GARD:0011989 {source="MONDO:equivalentTo"}
+xref: MESH:D058968 {source="MONDO:mondoIsNarrowerThanSource"}
+xref: UMLS:C0276912 {source="MONDO:mondoIsNarrowerThanSource"}
+is_a: MONDO:0005550 ! infectious disease
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-5002-8648
+property_value: RO:0002175 NCBITaxon:9606 {source="PMID:31845178", source="https://orcid.org/0000-0002-4142-7153", source="https://orcid.org/0000-0002-5002-8648"}
```

Comment:
Because this particular disease is only happening in human, you should be using an "in taxon" subclass of. 
But, since this term is in the human disease branch, you don't need it (it is implied)

Reply:
You might want to add the PMID as a dbxref for the definition though.

Reply:
Done!